salt/modules/pagerduty_util.py

```diff
@@ -0,0 +1,390 @@
+# -*- coding: utf-8 -*-
```

Comment:
TBH, I'm still not a fan of putting this here. Is this because of the portability and syncing thing with utils that @mgwilliams has been working on?